Plugin broken on IDEA 2026.2 EAP

### SQLDelight Version

2.3.2

### IDE Version

Build #IU-262.4852.50, built on May 7, 2026

### Dialect

SQLite

### Describe the Bug

It's broken!

### Stacktrace

```shell
java.lang.NoClassDefFoundError: org/jetbrains/kotlin/idea/util/projectStructure/ProjectStructureUtilKt
	at app.cash.sqldelight.intellij.ProjectService.findConfiguredFileIndex(ProjectService.kt:124)
	at app.cash.sqldelight.intellij.ProjectService.access$findConfiguredFileIndex(ProjectService.kt:63)
	at app.cash.sqldelight.intellij.ProjectService$1.after(ProjectService.kt:100)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeMethod(MessageBusImpl.kt:831)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeListener(MessageBusImpl.kt:775)
	at com.intellij.util.messages.impl.MessageBusImplKt.executeOrAddToQueue(MessageBusImpl.kt:596)
	at com.intellij.util.messages.impl.ToDirectChildrenMessagePublisher.publish$intellij_platform_core(CompositeMessageBus.kt:193)
	at com.intellij.util.messages.impl.MessagePublisher.invoke(MessageBusImpl.kt:544)
	at jdk.proxy2/jdk.proxy2.$Proxy162.after(Unknown Source)
	at com.intellij.openapi.vfs.newvfs.persistent.PersistentFSImpl.lambda$fireAfterEvents$4(PersistentFSImpl.java:1816)
	at com.intellij.openapi.application.impl.TransferredWriteActionServiceImpl.runOnEdtWithTransferredWriteActionAndWait$lambda$0(TransferredWriteActionServiceImpl.kt:13)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:114)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:95)
	at com.intellij.openapi.application.impl.InternalThreading.invokeAndWaitWithTransferredWriteAction$lambda$2(appImpl.kt:214)
	at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:193)
	at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:193)
	at com.intellij.util.concurrency.ChildContext.runInChildContext$lambda$1(propagation.kt:203)
	at com.intellij.util.concurrency.Propagation$runAsCoroutine$deferred$1.invokeSuspend(propagation.kt:448)
	at com.intellij.util.concurrency.Propagation$runAsCoroutine$deferred$1.invoke(propagation.kt)
	at com.intellij.util.concurrency.Propagation$runAsCoroutine$deferred$1.invoke(propagation.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startCoroutineUndispatched(Undispatched.kt:20)
	at kotlinx.coroutines.CoroutineStart.invoke(CoroutineStart.kt:360)
	at kotlinx.coroutines.AbstractCoroutine.start(AbstractCoroutine.kt:134)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.async(Builders.common.kt:88)
	at kotlinx.coroutines.BuildersKt.async(Unknown Source)
	at com.intellij.util.concurrency.Propagation.runAsCoroutine(propagation.kt:443)
	at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:203)
	at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:193)
	at com.intellij.util.concurrency.ContextRunnable.lambda$run$0(ContextRunnable.java:26)
	at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:294)
	at com.intellij.util.concurrency.ContextRunnable.run(ContextRunnable.java:25)
	at com.intellij.platform.locking.impl.NestedLocksThreadingSupport$transferWriteActionAndBlock$2.run(NestedLocksThreadingSupport.kt:1691)
	at com.intellij.openapi.application.impl.InternalThreading$TransferredWriteActionEvent$Companion.execute(appImpl.kt:287)
	at com.intellij.openapi.application.impl.InternalThreading$TransferredWriteActionEvent._init_$lambda$0(appImpl.kt:280)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:323)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:732)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:711)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:726)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:579)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0$0$0$0(IdeEventQueue.kt:383)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:924)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0$0$0(IdeEventQueue.kt:382)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$0(IdeEventQueue.kt:1116)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:106)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1116)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0(IdeEventQueue.kt:377)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:417)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.lang.ClassNotFoundException: org.jetbrains.kotlin.idea.util.projectStructure.ProjectStructureUtilKt PluginClassLoader(plugin=PluginMainDescriptor(name=SQLDelight, id=com.squareup.sqldelight, version=2.3.2, isBundled=false, path=~/Library/Application Support/JetBrains/IntelliJIdea2026.2/plugins/sqldelight-idea-plugin), packagePrefix=null, state=active, parents=ContentModuleDescriptor(id=intellij.kotlin.projectWizard.maven, package=org.jetbrains.kotlin.tools.projectWizard.maven) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.projectWizard.gradle, package=org.jetbrains.kotlin.tools.projectWizard.gradle) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.projectWizard.idea, package=org.jetbrains.kotlin.tools.projectWizard) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.projectWizard.compose, package=org.jetbrains.kotlin.tools.composeProjectWizard) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.onboarding.maven, package=org.jetbrains.kotlin.onboarding.maven) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.onboarding.gradle, package=org.jetbrains.kotlin.onboarding.gradle) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.onboarding, package=org.jetbrains.kotlin.onboarding) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.j2k, package=org.jetbrains.kotlin.j2k.k2) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.searching, package=org.jetbrains.kotlin.idea.searching) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.compilerReferenceIndex, package=org.jetbrains.kotlin.idea.search.refIndex) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.runConfigurations.jvm, package=org.jetbrains.kotlin.idea.runConfigurations.jvm) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.navigation, package=org.jetbrains.kotlin.idea.navigation) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.completion.ml, package=org.jetbrains.kotlin.idea.mlCompletion) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.compilerPlugins.support.maven, package=org.jetbrains.kotlin.idea.maven.compilerPlugin) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.maven, package=org.jetbrains.kotlin.idea.maven) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.lombok, package=org.jetbrains.kotlin.idea.lombok) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.codeInsight.liveTemplates, package=org.jetbrains.kotlin.idea.liveTemplates.k2) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=kotlin.code-insight.live-templates.shared, package=org.jetbrains.kotlin.idea.liveTemplates) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.refactorings.rename, package=org.jetbrains.kotlin.idea.k2.refactoring.rename) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.refactorings.move, package=org.jetbrains.kotlin.idea.k2.refactoring.move) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.refactorings, package=org.jetbrains.kotlin.idea.k2.refactoring) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.injection, package=org.jetbrains.kotlin.idea.k2.injection) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.jvm.debugger.evaluation, package=org.jetbrains.kotlin.idea.k2.debugger) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.codeInsight.structuralSearch, package=org.jetbrains.kotlin.idea.k2.codeinsight.structuralsearch) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.codeInsight.intentions, package=org.jetbrains.kotlin.idea.k2.codeinsight.intentions) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.codeInsight.inspections, package=org.jetbrains.kotlin.idea.k2.codeinsight.inspections) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.codeInsight.fixes, package=org.jetbrains.kotlin.idea.k2.codeinsight.fixes) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.codeInsight, package=org.jetbrains.kotlin.idea.k2.codeinsight) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.jvm.decompiler, package=org.jetbrains.kotlin.idea.jvmDecompiler) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=kotlin.jvm.shared, package=org.jetbrains.kotlin.idea.jvm.shared) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.jvm, package=org.jetbrains.kotlin.idea.jvm.k2) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.runConfigurations.junit, package=org.jetbrains.kotlin.idea.junit) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.internal, package=org.jetbrains.kotlin.idea.internal.actions) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.highlighting, package=org.jetbrains.kotlin.idea.highlighting) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.gradle.codeInsight.groovy, package=org.jetbrains.kotlin.idea.groovy) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.grazie, package=org.jetbrains.kotlin.idea.grazie) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.gradle.tooling, package=org.jetbrains.kotlin.idea.gradleTooling) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.gradle.java, package=org.jetbrains.kotlin.idea.gradleJava.k2) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=kotlin.gradle.gradle-java, package=org.jetbrains.kotlin.idea.gradleJava) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.gradle.codeInsight.common, package=org.jetbrains.kotlin.idea.gradleCodeInsightCommon) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.gradle.codeInsight.toml, package=org.jetbrains.kotlin.idea.gradle.versionCatalog.toml) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.gradle.gradle, package=org.jetbrains.kotlin.idea.gradle) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.git, package=org.jetbrains.kotlin.idea.git) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.coverage, package=org.jetbrains.kotlin.idea.coverage) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.base.scripting.shared, package=org.jetbrains.kotlin.idea.core.script.shared) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.base.scripting.scratch.definition, package=org.jetbrains.kotlin.idea.core.script.scratch.definition) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.base.scripting, package=org.jetbrains.kotlin.idea.core.script.k2) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.copyright, package=org.jetbrains.kotlin.idea.copyright) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.completion.impl, package=org.jetbrains.kotlin.idea.completion.impl.k2) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.compilerPlugins.scripting, package=org.jetbrains.kotlin.idea.compilerPlugin.scripting.gradleJava) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.compilerPlugins.samWithReceiver.maven, package=org.jetbrains.kotlin.idea.compilerPlugin.samWithReceiver.maven) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.compilerPlugins.noarg.maven, package=org.jetbrains.kotlin.idea.compilerPlugin.noarg.maven) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.compilerPlugins.lombok.maven, package=org.jetbrains.kotlin.idea.compilerPlugin.lombok.maven) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.compilerPlugins.serialization.maven, package=org.jetbrains.kotlin.idea.compilerPlugin.kotlinxSerialization.maven) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.compilerPlugins.serialization, package=org.jetbrains.kotlin.idea.compilerPlugin.kotlinxSerialization.k2) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.compilerPlugins.dataframe.maven, package=org.jetbrains.kotlin.idea.compilerPlugin.kotlinDataFrame.maven) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.compilerPlugins.kapt, package=org.jetbrains.kotlin.idea.compilerPlugin.kapt.gradleJava) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.compilerPlugins.assignment.maven, package=org.jetbrains.kotlin.idea.compilerPlugin.assignment.maven) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.compilerPlugins.assignment.fixes, package=org.jetbrains.kotlin.idea.compilerPlugin.assignment) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.compilerPlugins.allopen.maven, package=org.jetbrains.kotlin.idea.compilerPlugin.allopen.maven) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.codeInsight.postfixTemplates, package=org.jetbrains.kotlin.idea.codeInsight.postfix) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.codeInsight.lineMarkers, package=org.jetbrains.kotlin.idea.codeInsight.lineMarkers) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=kotlin.code-insight.intentions.shared, package=org.jetbrains.kotlin.idea.codeInsight.intentions.shared) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=kotlin.code-insight.inspections.shared, package=org.jetbrains.kotlin.idea.codeInsight.inspections.shared) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=kotlin.base.injection, package=org.jetbrains.kotlin.idea.base.injection) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.projectStructure, package=org.jetbrains.kotlin.idea.base.fir.projectStructure) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.codeInsight.base, package=org.jetbrains.kotlin.idea.base.fir.codeInsight) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.analysis.platform, package=org.jetbrains.kotlin.idea.base.fir.analysisApiPlatform) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.base.externalSystem, package=org.jetbrains.kotlin.idea.base.externalSystem) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.base.analysis.platform, package=org.jetbrains.kotlin.idea.base.analysisApiPlatform) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=kotlin.gradle.scripting.shared, package=org.jetbrains.kotlin.gradle.scripting.shared) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.gradle.scripting, package=org.jetbrains.kotlin.gradle.scripting.k2) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.kotlin.scripting, package=org.jetbrains.kotlin.base.fir.scripting) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=kotlin-ultimate.ultimate-native, package=com.jetbrains.mpp) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=kotlin-ultimate.native-debugger, package=com.jetbrains.kotlin.commonNative.debugger) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=kotlin-ultimate.common-native, package=com.jetbrains.kotlin.commonNative) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.debugger.jvm.advanced.kotlin, package=com.intellij.debugger.jvm.advanced.kotlin) <- PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), ContentModuleDescriptor(id=intellij.java.execution.impl.shared, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.compiler.instrumentationUtil.java8, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.compiler.instrumentationUtil, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.compiler.antTasks, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.platform.jps.build.javac.rt, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.frontback.impl, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.psi.impl, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.psi, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.frontback.psi.impl, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.frontback.psi, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.syntax, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.gradle.settings, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Gradle, id=com.intellij.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-plugin), ContentModuleDescriptor(id=intellij.gradle.toolingExtension, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Gradle, id=com.intellij.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-plugin), ContentModuleDescriptor(id=intellij.gradle.toolingExtension.impl, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Gradle, id=com.intellij.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-plugin), ContentModuleDescriptor(id=intellij.platform.vcs.log.graph) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.java.guiForms.compiler, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.platform.jps.build.dependencyGraph, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.debugger.impl.shared, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.execution.impl.frontend, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.copyright) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.ultimate.icons) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.debugger.memory.agent, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.uiDesigner, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.jsp, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.platform.uast, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.platform.uast.ide, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.analysis, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.jvm.analysis, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.jvm.analysis.refactoring, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.jvm.analysis.quickFix, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.uast, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.platform.vcs.log) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.vcs.dvcs) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.java.indexing, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.codeserver.core, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.indexing.impl, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.codeserver.highlighting, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.platform.jps.build, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.jam) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.analysis.impl, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.structureView, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.jsp.spi, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.execution, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.debugger, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.compiler, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.remoteServers) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.debugger.logpoints) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.debugger.impl.frontend, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.platform.structuralSearch) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.gradle.toolingProxy, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Gradle, id=com.intellij.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-plugin), ContentModuleDescriptor(id=intellij.gradle, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Gradle, id=com.intellij.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-plugin), ContentModuleDescriptor(id=intellij.gradle.properties) <- PluginMainDescriptor(name=Gradle, id=com.intellij.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-plugin), ContentModuleDescriptor(id=intellij.gradle.completion) <- PluginMainDescriptor(name=Gradle, id=com.intellij.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-plugin), ContentModuleDescriptor(id=intellij.gradle.completion.toml) <- PluginMainDescriptor(name=Gradle, id=com.intellij.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-plugin), ContentModuleDescriptor(id=intellij.xml.emmet) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.relaxng) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.java.langInjection.jps) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.platform.vcs.impl) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.java.vcs) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.platform.vcs.log.impl) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.vcs.dvcs.impl) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.spellchecker) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.java.impl, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.unscramble) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.structuralSearch) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.uast.ide, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.manifest, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.compiler.impl, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.langInjection) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.impl.refactorings, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.jvm.analysis.impl) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.debugger.impl, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.debugger.impl.backend, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.execution.impl, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.execution.impl.backend, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.terminal) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.ui, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.remoteServers.impl) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.impl.inspections, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.typeMigration, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.backend, loadingRule=REQUIRED) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.debugger.jvm.advanced.java) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.jam.impl) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.java.backend.split) <- PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), ContentModuleDescriptor(id=intellij.gradle.java.coverage) <- PluginMainDescriptor(name=Gradle for Java, id=org.jetbrains.plugins.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-java-plugin), ContentModuleDescriptor(id=intellij.gradle.dependencyUpdater) <- PluginMainDescriptor(name=Gradle for Java, id=org.jetbrains.plugins.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-java-plugin), ContentModuleDescriptor(id=intellij.gradle.java, loadingRule=EMBEDDED) <- PluginMainDescriptor(name=Gradle for Java, id=org.jetbrains.plugins.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-java-plugin), ContentModuleDescriptor(id=intellij.gradle.java.toml) <- PluginMainDescriptor(name=Gradle for Java, id=org.jetbrains.plugins.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-java-plugin), ContentModuleDescriptor(id=intellij.gradle.analysis) <- PluginMainDescriptor(name=Gradle for Java, id=org.jetbrains.plugins.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-java-plugin), ContentModuleDescriptor(id=intellij.gradle.java.groovy) <- PluginMainDescriptor(name=Gradle for Java, id=org.jetbrains.plugins.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-java-plugin), ContentModuleDescriptor(id=intellij.gradle.java.properties) <- PluginMainDescriptor(name=Gradle for Java, id=org.jetbrains.plugins.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-java-plugin), ContentModuleDescriptor(id=intellij.gradle.java.groovy.toml) <- PluginMainDescriptor(name=Gradle for Java, id=org.jetbrains.plugins.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-java-plugin), ContentModuleDescriptor(id=intellij.gradle.java.maven) <- PluginMainDescriptor(name=Gradle for Java, id=org.jetbrains.plugins.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-java-plugin), ContentModuleDescriptor(id=intellij.spellchecker.xml) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.gradle.java.groovy.ml) <- PluginMainDescriptor(name=Gradle for Java, id=org.jetbrains.plugins.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-java-plugin), ContentModuleDescriptor(id=intellij.platform.scriptDebugger.protocolReaderRuntime) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.scriptDebugger.backend) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.scriptDebugger.ui) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.tasks) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.tasks.impl) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.collaborationTools.auth.base) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.collaborationTools.auth) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.platform.collaborationTools) <- PluginMainDescriptor(name=IDEA CORE, id=com.intellij, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/lib), ContentModuleDescriptor(id=intellij.gradle.completion.kotlin) <- PluginMainDescriptor(name=Gradle, id=com.intellij.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-plugin), PluginMainDescriptor(name=Kotlin, id=org.jetbrains.kotlin, version=262.4852.50-IJ, package=org.jetbrains.kotlin, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/Kotlin), PluginMainDescriptor(name=Java, id=com.intellij.java, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/java), PluginMainDescriptor(name=Gradle, id=com.intellij.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-plugin), PluginMainDescriptor(name=Gradle for Java, id=org.jetbrains.plugins.gradle, version=262.4852.50, isBundled=true, path=~/Applications/IntelliJ IDEA EAP.app/Contents/plugins/gradle-java-plugin), )
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.kt:154)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:490)
	... 53 more
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -25,6 +25,7 @@
 - [Intellij Plugin] IDE plugin can perform code completions for all dialects (#6210 by @griffio)
 - [Gradle Plugin] Fix circular dependency error running verify database task (#6221 by @griffio)
 - [Compiler] Fix optimistic lock for multirow update (#6240 by @griffio)
+- [Intellij Plugin] Fix deprecations causing crash in IDEA 2026.2 (#6247 by @griffio)
 
 ## [2.3.2] - 2026-03-16
 [2.3.2]: https://github.com/sqldelight/sqldelight/releases/tag/2.3.2

--- a/sqldelight-idea-plugin/src/main/kotlin/app/cash/sqldelight/intellij/ProjectService.kt
+++ b/sqldelight-idea-plugin/src/main/kotlin/app/cash/sqldelight/intellij/ProjectService.kt
@@ -19,11 +19,9 @@ import app.cash.sqldelight.core.SqlDelightFileIndex
 import app.cash.sqldelight.core.SqlDelightProjectService
 import app.cash.sqldelight.core.compiler.SqlDelightCompiler
 import app.cash.sqldelight.core.lang.MIGRATION_EXTENSION
-import app.cash.sqldelight.core.lang.MigrationFileType
 import app.cash.sqldelight.core.lang.MigrationParserDefinition
 import app.cash.sqldelight.core.lang.SQLDELIGHT_EXTENSION
 import app.cash.sqldelight.core.lang.SqlDelightFile
-import app.cash.sqldelight.core.lang.SqlDelightFileType
 import app.cash.sqldelight.dialect.api.SqlDelightDialect
 import app.cash.sqldelight.dialect.api.TypeResolver
 import app.cash.sqldelight.intellij.gradle.FileIndexMap
@@ -39,6 +37,7 @@ import com.intellij.openapi.Disposable
 import com.intellij.openapi.application.ApplicationManager
 import com.intellij.openapi.diagnostic.Logger
 import com.intellij.openapi.module.Module
+import com.intellij.openapi.module.ModuleUtilCore
 import com.intellij.openapi.project.Project
 import com.intellij.openapi.roots.ProjectRootManager
 import com.intellij.openapi.vfs.VirtualFile
@@ -57,7 +56,6 @@ import java.io.PrintStream
 import java.nio.file.Path
 import java.nio.file.Paths
 import kotlin.reflect.jvm.jvmName
-import org.jetbrains.kotlin.idea.util.projectStructure.getModule
 import timber.log.Timber
 
 class ProjectService(val project: Project) :
@@ -121,7 +119,7 @@ class ProjectService(val project: Project) :
   }
 
   private fun VirtualFile.findConfiguredFileIndex(): SqlDelightFileIndex? {
-    val module = getModule(project) ?: return null
+    val module = ModuleUtilCore.findModuleForFile(this, project) ?: return null
     val fileIndex = SqlDelightFileIndex.getInstance(module)
     return fileIndex.takeIf { it.isConfigured }
   }

--- a/sqldelight-idea-plugin/src/main/kotlin/app/cash/sqldelight/intellij/SqlDelightGotoDeclarationHandler.kt
+++ b/sqldelight-idea-plugin/src/main/kotlin/app/cash/sqldelight/intellij/SqlDelightGotoDeclarationHandler.kt
@@ -27,6 +27,7 @@ import com.alecstrong.sql.psi.core.psi.SqlStmt
 import com.intellij.codeInsight.navigation.actions.GotoDeclarationHandler
 import com.intellij.openapi.actionSystem.DataContext
 import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.module.ModuleUtilCore
 import com.intellij.openapi.progress.EmptyProgressIndicator
 import com.intellij.openapi.progress.ProgressManager
 import com.intellij.openapi.project.DumbService
@@ -42,7 +43,6 @@ import com.intellij.psi.search.GlobalSearchScope
 import com.intellij.psi.search.searches.ReferencesSearch
 import com.intellij.psi.util.childrenOfType
 import org.jetbrains.kotlin.idea.references.KtSimpleNameReference
-import org.jetbrains.kotlin.idea.util.projectStructure.getModule
 import org.jetbrains.kotlin.psi.KtNameReferenceExpression
 import org.jetbrains.kotlin.psi.KtNamedFunction
 import org.jetbrains.kotlin.psi.KtParameter
@@ -69,7 +69,7 @@ class SqlDelightGotoDeclarationHandler : GotoDeclarationHandler {
     val function = targetData.function
     val elementFile = targetData.containingFile
 
-    val module = elementFile?.getModule(function.project) ?: return emptyArray()
+    val module = elementFile?.let { ModuleUtilCore.findModuleForFile(it, function.project) } ?: return emptyArray()
 
     // Only handle files under the generated sqlite directory.
     val fileIndex = SqlDelightFileIndex.getInstance(module)

--- a/sqldelight-idea-plugin/src/main/kotlin/app/cash/sqldelight/intellij/actions/file/SqlDelightCreateFileAction.kt
+++ b/sqldelight-idea-plugin/src/main/kotlin/app/cash/sqldelight/intellij/actions/file/SqlDelightCreateFileAction.kt
@@ -18,7 +18,6 @@ import com.intellij.openapi.actionSystem.DataContext
 import com.intellij.openapi.application.runWriteAction
 import com.intellij.openapi.editor.LogicalPosition
 import com.intellij.openapi.fileEditor.FileEditorManager
-import com.intellij.openapi.module.ModuleUtilCore
 import com.intellij.openapi.module.ModuleUtilCore.findModuleForFile
 import com.intellij.openapi.project.DumbAware
 import com.intellij.openapi.project.Project
@@ -28,7 +27,6 @@ import com.intellij.psi.PsiFile
 import com.intellij.psi.util.PsiTreeUtil
 import com.intellij.util.IncorrectOperationException
 import java.util.Properties
-import org.jetbrains.kotlin.idea.util.findModule
 
 /**
  * Creates a new SqlDelight file/table/migration from a template (see [fileTemplates.internal])

